src/composables/useSession/useSession.ts

```diff
@@ -27,10 +40,37 @@ export default function useSession() {
         });
     };
 
+    /**
+     * Generates a new session token with uri encoding
+     */
+    const generateSessionToken = (): void => {
+        sessionToken.value = generateValidToken();
+        encodeSessionToken();
+    };
+
+    /**
+     * Encodes the session token if it is not already encoded by default
+     */
+    const encodeSessionToken = (): void => {
```

Comment:
'%' is actuall not valid itself (this function seems to not encode this correctly)

Reply:
<img width="590" alt="image" src="https://github.com/VisualGameData/VIGAD/assets/70775490/10305968-a986-41c8-858c-054ce8442cbc">

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent

idk what the workaround could be, a single % does not really work, or atleast there is not an elegant solution

Reply:
that snipped seems like a good direction. The thing is that % as itself is not URI conform, however %D1 for example is (that is the encoding for some special character)

Reply:
Yeah i think i just need to inform the user then that the provided input is not URI conform basically